plaidml/op/BUILD

```diff
@@ -63,8 +64,15 @@ py_library(
 plaidml_cc_test(
     name = "cc_test",
     srcs = ["op_test.cc"],
+    tags = ["manual"],  # the `edsl_test.cc.test` rule will run this test via lit
```

Comment:
If we keep the non-lit aspects of the tests (e.g. the `runProgram(program);` lines) we should not mark this `manual`, as both tests will be needed.

Reply:
So just remove this line? Or should it be changed to something else?

Reply:
No, it needs to be marked as manual, because lit will run it already.

Reply:
thanks for the correction!